backend/cms/models/page.py

```diff
@@ -66,7 +66,8 @@ def languages(self):
 
     def get_translation(self, language_code):
         try:
-            page_translation = self.page_translations.get(language__code=language_code)
+            page_translation = self.page_translations.filter(
+                language__code=language_code).order_by('-version')[0]
```

Comment:
Do we prefer `.order_by('-version')[0]` or `.order_by('-version').first()` ?

Reply:
I personally prefer the latter

Reply:
I agree with Flo.